base.rb

```diff
@@ -0,0 +1,48 @@
+require 'bundler'
+require 'yaml'
+require 'sinatra/base'
+require "sinatra/multi_route"
+require 'securerandom'
+require 'active_support/core_ext/hash'
+require 'active_support/inflector'
+require './lib/tendrl'
```

Comment:
Initially had used just `require 'tendrl'` assuming the server loads the lib folder automatically, but that needed me to add -Ilib everytime, so just added the require relative instead.